docs/faq/ImmutableData.md

```diff
@@ -291,7 +291,7 @@ Note that, conversely, if the values in your props object refer to mutable objec
 ## What approaches are there for handling data immutably? Do I have to use Immutable.js? 
 You do not need to use Immutable.JS with Redux. Plain JavaScript, if written correctly, is perfectly capable of providing immutability without having to use an immutable-focused library. 
 
-However, guaranteeing immutability with JavaScript is difficult, and it can be easy to mutate an object accidentally, causing bugs in your app that are extremely difficult to locate. For this reason, using a library such as Immutable.JS can significantly improve the reliability of your app, and make your app’s development much easier.
+However, guaranteeing immutability with JavaScript is difficult, and it can be easy to mutate an object accidentally, causing bugs in your app that are extremely difficult to locate. For this reason, using an immutable update utility library such as Immutable.JS can significantly improve the reliability of your app, and make your app’s development much easier.
```

Comment:
I wouldn't call Immutable.js an "immutable update utility library".  It's a full-blown set of specialized data structures.  There's really three categories of JS immutable libs: specialized data structures (Immutable.js, Mori); libs that freeze JS objects (seamless-immutable); libs that abstract out the update process (dot-prop-immutable, etc)